documentation/general/dotnet-run-file.md

```diff
@@ -162,20 +153,24 @@ The generated folders might need to be named differently to avoid clashes with e
 
 The entry-point projects (`Program1` and `Program2` in our example)
 have the shared `.cs` files source-included via `<Content Include="../Shared/**/*.cs" />`.
-We could consider having the projects directly in the top-level folder instead
-but that might result in clashes of build outputs that are not project-scoped, like `project.assets.json`.
-If we did that though, it would be enough to exclude the other entry points rather than including all the shared `.cs` files.
 
-Unless the [artifacts output layout][artifacts-output] is used (which is recommended),
-those implicit projects mean that build artifacts are placed under those implicit directories
-even though they don't exist on disk prior to build:
-```
-App/Program1/bin/
-App/Program1/obj/
-App/Program2/bin/
-App/Program2/obj/
+We could consider using `InternalsVisibleTo` attribute but that might result in slight differences between single- and multi-entry-point programs
```

Comment:
Not sure if we want to use InternalsVisibleTo or source sharing. Went for source sharing in this doc but can change.